scripts/export_sqlite_to_workspace.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import os
+import sys
+import argparse
+import sqlite3
+import json
+import uuid
+import shutil
+
+import bcrypt
+
+
+def export_db(database=None, workspace=None, openplc=None):
+
+    ## Connect db
+    if not os.path.exists(database):
+        print("ERROR: No database at '%s'" % database)
+        return
+    try:
+        conn = sqlite3.connect(database)
+        cursor = conn.cursor()
+        cursor.execute("select 1")
+    except sqlite3.Error as e:
+        print(e)
+        return
+
+    ## Validate workspace
+    workspace = os.path.abspath(workspace)
+    if not os.path.exists(workspace):
+        print("ERROR: No workspace folder at `%s`" % workspace)
+        return
+    if not os.path.isdir(workspace):
+        print("ERROR: File at `%s` instead of a folder" % workspace)
+        return
+
+    ## Validate openplc
+    openplc = os.path.abspath(openplc)
+    if not os.path.exists(openplc):
+        print("ERROR: No openplc at  `%s`" % openplc)
+        return
+
+
+
+    def query_db(sql, args=()):
```

Comment:
Any reason that this is defined as a nested function? You can make this at the module scope so that someone else could use it if the so desired. This way, it is inaccessible